CONTRIBUTING

```diff
@@ -0,0 +1,9 @@
+How to contribute to MPICH
+
+1. Sign and subnit a contributors' agreement. http://www.mpich.org/documentation/contributor-docs/
```

Comment:
submit

Reply:
You might want to consider something like this: https://cla-assistant.io

I assume this is similar to what libfabric uses (if not the same thing).